contracts/token/ERC721/README.md

```diff
@@ -26,12 +26,17 @@ This set of interfaces, contracts, and utilities are all related to the [ERC721
 
 *For a walkthrough on how to create an ERC721 token read our [ERC721 guide](../../tokens.md#erc721).*
 
-The EIP consists of three interfaces, found here as `IERC721`,
-`IERC721Metadata`, and `IERC721Enumerable`. Only the first one is required in a
-contract to be ERC721 compliant. Each interface is implemented separately in
-`ERC721`, `ERC721Metadata`, and `ERC721Enumerable`. You can choose the subset
-of functionality you would like to support in your token by combining the
-desired subset through inheritance. The fully featured token implementing all
-three interfaces is prepackaged as `ERC721Full`.
+The EIP consists of three interfaces, found here as `IERC721`, `IERC721Metadata`, and `IERC721Enumerable`. Only the first one is required in a contract to be ERC721 compliant.
+
+Each interface is implemented separately in `ERC721`, `ERC721Metadata`, and `ERC721Enumerable`. You can choose the subset of functionality you would like to support in your token by combining the
+desired subset through inheritance.
+
+The fully featured token implementing all three interfaces is prepackaged as `ERC721Full`.
+
+Additionally, `IERC721Receiver` can be used to prevent tokens from becoming forever locked in contracts. Imagine sending an in-game item to an exchange address that can't send it back!. When using `safeTransferFrom()`, the token contract checks to see that the receiver is an `IERC721Receiver`, which implies that it knows how to handle `ERC721` tokens. If you're writing a contract that needs to receive `ERC721` tokens, you'll want to include this interface.
```

Comment:
Perhaps the very first sentence, but the actual behavior is clearly expressed immediately after, I think. What would you change that sentence for?

Reply:
How about

> If you're developing a contract that receives ERC721 tokens, it has to implement the `IERC721Receiver` interface. The reason is that `safeTransferFrom`, which users and applications are encouraged to use, will reject to transfer to a contract that doesn't implement this interface. This is meant to prevent tokens from becoming accidentally locked in contracts. Imagine sending an in-game item to an exchange address that can't send it back! When using `safeTransferFrom()`, the token contract checks to see that the receiver is an `IERC721Receiver`, which implies that it knows how to handle `ERC721` tokens.